Azure-Samples/java-on-azure-examples
# Fix badges

## Purpose

* Fixes #143

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  The work flow badges should be working now (either passing or failing, but correctly displayed in READMEs

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
mvn clean install
```

## What to Check
Verify that the following are valid
* ...

## Other Information
